wgsl/keywords

```diff
@@ -509,6 +509,8 @@ while
 texture
 sampler
 );
+
+# https://github.com/gpuweb/gpuweb/pull/3615 Don't reserve: line lineadj point
```

Comment:
We probably don't need this comment in source file, but PR can be coauthored. Thanks.

Reply:
I am synced and coauthored your addition in #3615  💅